shared/fs/footer/upload.desktop.js

```diff
@@ -37,26 +38,46 @@ const realCSS = `
 }
 `
 
-const Upload = ({showing, files, totalSyncingBytes, timeLeft, debugToggleShow}: UploadProps) => (
-  <React.Fragment>
-    {!!debugToggleShow && <Button type="Primary" onClick={debugToggleShow} label="Toggle" />}
-    <CSSTransition in={showing} classNames="upload-animation" timeout={300} unmountOnExit={true}>
-      <Box className="upload-animation-loop" style={stylesBox}>
-        <style>{realCSS}</style>
-        <Text key="files" type="BodySemibold" style={stylesText}>
-          {files
-            ? `Encrypting and uploading ${files} files...`
-            : totalSyncingBytes
-              ? 'Encrypting and uploading...'
-              : 'Done!'}
-        </Text>
-        {!!(timeLeft && timeLeft.length) && (
-          <Text key="left" type="BodySmall" style={stylesText}>{`${timeLeft} left`}</Text>
-        )}
-      </Box>
-    </CSSTransition>
-  </React.Fragment>
-)
+const styles = Styles.styleSheetCreate({
+  textOverflow: Styles.platformStyles({
+    common: {
+      color: globalColors.white,
+      maxWidth: '60%',
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textAlign: 'center',
+      textOverflow: 'ellipsis',
+    },
+    isElectron: {
+      overflowWrap: 'ellipsis',
+    },
+  }),
+})
```

Comment:
Usually we put the styles on the bottom of the page, before the exports. That is primarily so that a reader of the code sees relevant code when they open a file, rather than a bunch of styles.